### ISSUE TITLE
Electron 5?

### ISSUE THREAD

**Issue by asgarli** [BenjaminDobler/ngtron#6]
I noticed that the current Electron version is 4. I changed package version to 5 and it seems to work, so maybe it's worth updating electron 5 in the schematics?

**Comment by BenjaminDobler** [comment]
Yeah in the end i might let the user choose when adding ngtron becuase there are some quite fundamental changes from electron 4 to 5 but yeah right now it would probably best to use the latest version by default.

**Comment by chadbr** [comment]
Electron 6? :)
